### ISSUE TITLE
Running 1.7 cpu profiling on Azul Zing's JVM crashes with SIGTERM

### ISSUE THREAD

**Issue by sternr** [jvm-profiling-tools/async-profiler#318]
Crash details:

```
0: rip=0x00007fa770c46132 @rip=[0xffffffffffffffff] (hotspot_os_backtrace_callback.part.39+34) [gcc frame, calls gcc]
1: rip=0x00007fa76fbced1f @rip=[0x00007fa3fc3e8d88] (os_backtrace+31) [gcc frame, calls gcc]
2: rip=0x00007fa770c454cd @rip=[0x00007fa3fc3e8df8] (jvm_unexpected_exception_handler+205) [gcc frame, calls gcc]
3: rip=0x00007fa770c4551f @rip=[0x00007fa3fc3e8ea8] (jvm_unexpected_exception_helper+15) [gcc frame, calls gcc]
4: rip=0x00007fa76fbce70b @rip=[0x00007fa3fc3e8eb8] (jvm_unexpected_exception_handler_wrapper+107) [gcc frame, calls gcc]
#
# An unexpected error has been detected by Java Runtime Environment:
#
#  Segmentation fault (0xb) at pc=0x7f9ce8cfe8a0, pid=266618, tid=267848
#
# Java VM: Zing 64-Bit Tiered VM (1.8.0-zing_19.01.0.0-b6-product-azlinuxM-X86_64, mixed mode)
# Problematic frame:
# C  [libasyncProfiler.so+0x218a0]  Profiler::getJavaTraceAsync(void*, ASGCT_CallFrame*, int)+0xe0
```

**Comment by sternr** [comment]
We've been using 1.5 for a while now with no issues, so it's something between 1.5 and 1.7

**Comment by sternr** [comment]
Ok seems to be fixed in 1.7.1, thanks!
